datafusion/physical-plan/src/projection.rs

```diff
@@ -147,6 +147,8 @@ impl ProjectionExec {
     }
 }
 
+pub type ProjectionExpr = (Arc<dyn PhysicalExpr>, String);
```

Comment:
I recommend a future PR transform this into a struct with named fields. That will be a breaking change so I am not going to do it in this PR, it should be done in isolation.